package.json

```diff
@@ -128,5 +128,12 @@
     "test-cov": "nyc --require=flow-remove-types/register --reporter=text-summary --reporter=lcov --cache run-s test-unit test-render test-query",
     "prepublish": "in-publish && run-s build-dev build-min || not-in-publish",
     "postinstall": "cd src/style-spec && yarn || npm install"
-  }
+  },
+  "files": [
+    "src/",
+    "dist/",
+    "plugins/",
+    "LICENSE.txt",
+    "ARCHITECTURE.md"
```

Comment:
Readme and changelog are automatically added see https://docs.npmjs.com/files/package.json#files